cmd/global-heal.go

```diff
@@ -72,11 +72,11 @@ func getLocalBackgroundHealStatus() madmin.BgHealState {
 	if !ok {
 		return madmin.BgHealState{}
 	}
-
+	// FIXME:
```

Comment:
Any good ideas what to return here?

Reply:
We could return 

- the number of objects successfully healed
- the number of objects failed to heal / couldn't be recovered
- the number of objects in dangling state / removed

Reply:
Yeah. Main thing is that is will now be distributed across all severs instead of just the one that has the background job. I guess when it will make sense to add the prometheus numbers across servers when those are added.